queries/cpp/highlights.scm

```diff
@@ -19,6 +17,9 @@
 (optional_parameter_declaration
     declarator: (_) @parameter)
 
+((identifier) @parameter
+
+  (#has-ancestor? @parameter "parameter_declaration"))
```

Comment:
Meant this one. It's already in C highlights

Reply:
Oh yeah sorry huh